TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md

```diff
@@ -0,0 +1,171 @@
+
+* Bokdeuk Jeong (Samsung)
+* Chris Ramming (Broadcom)
+* Dan Willians (Intel)
+* Drasko Draskovic (Ultraviolet)
+* Eric Voit (Cisco)
+* Jiewen Yao (Intel)
+* Jorg Rodel (SUSE)
+* Kanth Ghatraju (Oracle) 
+* Ken Lu (Intel)
+* Marc Meunier (Arm)
+* Mark Novak (JP Morgan Chase)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Samuel Ortiz (Rivos)
+* Sal Kimmich (GadflyAI)
+* Simon Frost (Arm)
+* Sung Lee (Broadcom)
+* Wenhui Zhang (Bytedance/TikTok)
+* Yash Mankad (Red Hat)
+* Ye Li (Broadcom)
+
+* Ben Sternthal (LF)
+* Riaan Kleinhans (LF)
+
+
+## Welcome New Community Members
+Sal Kimmich (CCC)
+Dan Williams (Intel)
+
+New community members introduced themselves. Sal Kimmich introduced themselves and stated that they would be starting as the new community manager on Monday.
+
+
+## Meeting Minute Approval
+
+Dan Middleton (DM) reviewed the minutes of the last meeting and stated that we would experiment with approving minutes via Github.
+Going forward meeting minutes will be approved according to the GB agreement where a PR can be merged with two approvals.
+
+
+## Scaling Project Mentors 
+
+DM spoke to three options for scaling project mentorship. The first option is assigning voting members round-robin style. The second option would be to stop doing this entirely. The third option would be looking for common topic areas among the projects, and TAC folks volunteer to be mentors for those topics. 
+
+Chris Ramming (CR) expressed support for option three and stated that one of the most important mentorship responsibilities for a mentor is finding connections between projects. 
+
+Yash Mankad (YM) asked if any mentors on the call could give feedback on their experience but none was in the meeting.
+
+Catalin Sandu (CS) raised concerns about time commitments that members might not be able to manage. DM responded that member involvement with the project is important.
+
+Sal Kimmich (SM) Pointed to developer inventions and the benefits to the communities where mentorship support project improvements.Process patch as opposed to code patches.
+
+Create Linux Kernel SIG #196Chris Ramming raises the topic of project’s Github repository management and management of repos under a single org. - CR would create an Github issue to outline this.
+
+[Address shortage of Project Mentors #207](https://github.com/confidential-computing/governance/issues/207)
+
+## Project administration 
+
+Benjamin Sternthal (BS) spoke about optimizing the project progression process and eliminating projects, creating a charter as a first step. BS also shared the [tracking dashboard](https://github.com/orgs/confidential-computing/projects/2/views/1) they are now using for projects.
+
+Riaan Kleinhans (RK) introduced the LF Voting tool available for the TAC and other committees. Riaan or Ben can create and manage the votes as it is shared with them.
+
+
+
+## Project Proposal Coconut SVSM 
+
+Jorg Rodel (JR) proceeds to answer technical questions about the Coconut SVCM project.
```

Comment:
Jorg's last name is spelled Roedel

https://www.suse.com/c/author/joro/

Reply:
Also ... 'Coconut SVSM'

Reply:
Updated